### PR TITLE
create register form

### PR DESCRIPTION
**Pull request by euniceyoh**

### CODE REVIEW

**Comment by christinang3030** [comment]
I don't see any problems with the login.
Is there a way we can test the registration form?

**Review by charlotte-zhuang — APPROVED** [review]
This will need to be updated after #28 is merged.

File: src/components/Login.jsx [hunk 1]
```diff
@@ -9,25 +9,32 @@ function Login() {
         ? JSON.parse(localStorage.getItem('loginData'))
         : null
     );
+
     const handleFailure = (result) => {
         console.log(result);
     }
+
+    console.log("test")
```
Comment (JoshuaLawson8) [inline src/components/Login.jsx]: leave in?

File: src/components/Register.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
```
Comment (JoshuaLawson8) [inline src/components/Register.jsx:1]: don't need to import react

**Review by JoshuaLawson8 — APPROVED** [review]
Looks good

**Pull request closed without merging**